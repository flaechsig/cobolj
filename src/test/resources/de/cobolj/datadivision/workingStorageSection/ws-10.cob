IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-ID   PIC S9(10) VALUE -12345.
   01 WS-DESC PIC  A(10) VALUE "ID-NAME: ".

PROCEDURE DIVISION.
   DISPLAY WS-DESC WS-ID.
STOP RUN.