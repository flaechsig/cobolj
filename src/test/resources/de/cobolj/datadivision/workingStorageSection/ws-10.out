ID-NAME:  -0000012345
